model-server-openapi/specifications/model-server-html.yaml

```diff
@@ -37,7 +37,7 @@ paths:
               examples:
                 Example#1:
                   value: "../repos/"
-  /content/{repository}/{branch}/latest:
+  /content/repositories/{repository}/branches/{branch}/latest:
```

Comment:
These changes should be marked as breaking changes.

Reply:
These endpoints are only used by the content explorer which is part of the model-server. I don't know why there is an Open API specification of it. I don't consider this a breaking change.

Reply:
oh yes, missed that this is the content explorer file, which should be deleted in any case.